tools/version.sh

```diff
@@ -74,7 +74,7 @@ get_version_vars() {
 
     # Use git describe to find the version based on tags.
     #if [[ -n ${KUBE_GIT_VERSION-} ]] || KUBE_GIT_VERSION=$("${git[@]}" describe --tags --match='v*' --abbrev=14 "${KUBE_GIT_COMMIT}^{commit}" 2>/dev/null); then
-    if [[ -n ${KUBE_GIT_VERSION-} ]] || KUBE_GIT_VERSION=$(echo "v1.18.0-alpha+${KUBE_GIT_COMMIT}"); then
+    if [[ -n ${KUBE_GIT_VERSION-} ]] || KUBE_GIT_VERSION=$(echo "v1.20.0+${KUBE_GIT_COMMIT}"); then
```

Comment:
Is there a reason we removed the "-alpha" suffix? I've generally found it useful to indicate a local build.